tgui/packages/tgui/interfaces/PreferencesMenu/GamePreferencesPage.tsx

```diff
@@ -210,6 +223,15 @@ export const GamePreferencesPage = (props, context) => {
         <Flex.Item grow>
           <Input autoFocus fluid placeholder="Search options" value={searchText} onInput={(_, value) => setSearchText(value)} />
         </Flex.Item>
+        <Flex.Item>
+          <ButtonCheckbox
+            checked={advancedSettings}
+            onClick={() => {
+              setAdvancedSettings(!advancedSettings);
+            }}>
+            Show Advanced Settings
```

Comment:
I'm unhappy with how this looks. The button is pretty large, and it decenters the very intentionally placed searchbar. It might have some place in the titlebar of the window. I think if you reduce the length of the text in here (maybe just use an icon instead?) and add some margins around it, fix the centering of the searchbar.. it should be OK.

Reply:
An icon would probably be confusing, I could probably reduce the text but I think this is the most appropriate space for it. It has a fixed width so doesn't cause elements of the UI to jump around and cause confusion and handles scaling nicely.

I can move it, but I think putting it in the titlebar wouldn't be an improvement since it would be away from the rest of the elements that do the same thing and would ruin the quite nice heirarchy that the UI has going on where from top to bottom it gets more refined
- Player vs character prefs
- Settings vs Keybinds
- Filters
- Links
- Actions

Reply:
I'll try reducing the length of the text

Reply:
![image](https://github.com/user-attachments/assets/09644995-834c-4675-a6ba-b4b78cf04b9e)

To be honest, it doesn't really make much of a difference. The search bar is big enough anyway and I think this is a fine location

Reply:
Much better with less text